### DISCUSSION TITLE
novelwriter disappeared from my system

### DISCUSSION THREAD

**Discussion by AKPhoto** [vkbo/novelWriter#771] [Help]
Got to my Linux Ubuntu 20.04 laptop today and wanted to continue with my novel. got the following message... "Could not find the program '/home/andrew/Downloads/novelWriter-1.2.3-minimal-linux/novelWriter.py'"
Tried reinstalling and restarting laptop, now novelwriter is just not starting up. Not sure what is going on. Novel was about a hundred pages in hope it isn't gone entirely.

**Comment by vkbo** [comment]
Fist of all, you probably shouldn't keep an application you're using in the Downloads folder. Copy it to somewhere that is less at risk for being accidentally deleted during a cleanup.

I'm not sure which install steps you used, but it looks like you're running it from the minimal package. If you just ran the `xdg-install` command, all it does is create the launcher and icons and add a definition file to be able to double-click a project to open it. You can update that launcher by running `./setup.py xdg-install` again from the folder where you have the novelWriter files extracted.

If novelWriter doesn't start, you can try to start it directly from the folder where you extracted it by running `./novelWriter.py` in the terminal. Then it will tell you the reason it cannot start.

As for the project, installing or uninstalling novelWriter does not affect your projects. Unless something else has happened, your novel should be fine. Although please keep backups!

**Reply by vkbo** [reply]
Hi again, I'm happy to help you with this, but please respond in this topic, and not in other discussions. Thanks!
